ingenialink/canopen/network.py

```diff
@@ -8,10 +8,16 @@
 from time import sleep
 from typing import Any, Callable, Dict, List, Optional, Tuple, Union
 
+RUNNING_ON_WINDOWS = os.name == "nt"
```

Comment:
We could use the [system](https://docs.python.org/3/library/platform.html#platform.system) method to check the OS.

Reply:
Done [9d6f708](https://github.com/ingeniamc/ingenialink-python/pull/390/commits/9d6f708707fbcf386670c5534ef19ed948f7e0bb)